style.css

```diff
@@ -0,0 +1,825 @@
+}
+
+.about-page-section .contact-us-text {
+  text-align: center;
+  margin-top: 30px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 60%;
+  font-size: 2.5vh;
+  color: #545454;
+  line-height: 30px;
+}
+
+.about-page-section a {
+  color: #272a31;
+  font-weight: 600;
+}
+
+.our-logo {
+  padding-top: 40px;
+  padding-bottom: 30px;
+}
+
+.our-logo .our-logo-header {
+  position: static;
+  padding: 0;
+  text-align: center;
+}
+
+.our-logo .our-logo-header h2 {
+  color: #272a31;
+}
+
+.our-logo .our-logo-header .our-logo-hr {
+  width: 30px;
+  margin: 15px auto;
+  border-color: #ec5242;
+}
+
+.our-logo .our-logo-body {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.our-logo .our-logo-body p {
+  font-size: 2.5vh;
+  color: #545454;
+  line-height: 30px;
+}
+
+.our-logo .our-logo-body img {
+  width: 100%;
+  padding: 30px;
+  border: 1px solid #d3d3d3;
+  box-sizing: border-box;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+#logo-text-desktop {
+  display: none;
+}
+
+.previous-events {
+  padding-top: 40px;
+  padding-bottom: 40px;
+  border-top: 1px solid #d3d3d3;
+}
+
+.previous-events .container {
+  text-align: center;
+}
+
+.previous-events .container .previous-events-header {
+  position: static;
+  padding: 0;
+  text-align: center;
+}
+
+.previous-events .container .previous-events-header h2 {
+  color: #272a31;
+}
+
+.previous-events .container .previous-events-header .our-logo-hr {
+  width: 30px;
+  margin: 15px auto;
+  border-color: #ec5242;
+}
+
+.previous-events .container p {
+  color: #545454;
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+.previous-events .container .previous-events-body {
+  text-align: center;
+}
+
+.previous-events .container .previous-events-body .previous-event-img-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  background-color: rgba(255, 79, 56, 0.9);
+  background-image: url(assets/2022_event.jpeg);
+  width: 80%;
+  height: 280px;
+  background-position: center;
+  background-size: cover;
+  background-blend-mode: multiply;
+  text-decoration: none;
+}
+
+.previous-events .container .previous-events-body .previous-event-img-container p {
+  color: white;
+  font-size: 4vh;
+}
+
+.previous-events .container .previous-events-body .previous-event-img-container p strong {
+  font-weight: 800;
+  font-size: 5vh;
+}
+
+.previous-events .container .previous-events-body .prev-second-img {
+  background-image: url(assets/2021_event.jpeg);
+  margin-top: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+    margin: 0;
+  }
+
+  .toolbar {
+    display: block;
+  }
+
+  .nav-menu-container {
+    display: flex;
+  }
+
+  .welcome {
+    margin-top: 106px;
+    height: calc(100vh - 106px);
+  }
+
+  .welcome .welcome-container {
+    text-align: left;
+    width: 100%;
+  }
+
```

Comment:
- [ ] Great job so far. However, in the design, this section has spacing before and after the section, please fix that.


### Your Implementation

![image](https://user-images.githubusercontent.com/59416136/163163547-6f09dd6d-02f6-4f48-bddc-b9c7559892e3.png)

### Behance Design

![image](https://user-images.githubusercontent.com/59416136/163163686-e958016b-798e-4eaf-ae5e-87a49657c0c0.png)